app/helpers/middleware_server_helper/textual_summary.rb

```diff
@@ -0,0 +1,55 @@
+module MiddlewareServerHelper::TextualSummary
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    @prop_hash = eval(@record.properties)
```

Comment:
Ah ok...but instead of eval, then you can do JSON.parse, which is *much* safer.

In addition, the UI should not be aware of the content of columns in this fashion...it's much too provider specific.  The knowledge that this is JSON content belongs with the parser.  Thus, what I think is the best is to make properties a serialized column (i.e. make it text column type and in the model declare it as `serialize :properties`), then in the parser when you set the column do `:properties => JSON.parse(eap.properties)`